qutebrowser/browser/webengine/spell.py

```diff
@@ -21,26 +21,46 @@
 
 import glob
 import os
+import re
 
+from qutebrowser.utils import log
```

Comment:
Please put this after the PyQt5 import with a blank line in between, so it's grouped in stdlib/other/qutebrowser imports (see [pep8](https://www.python.org/dev/peps/pep-0008/#imports))